src/eduid/webapp/common/session/logindata.py

```diff
@@ -273,6 +327,15 @@ def get_requested_authn_context(self) -> Optional[EduidAuthnContextClass]:
         """
         return _pick_authn_context(self.authn_contexts, self.request_ref)
 
+    @property
+    def service_requested_eppn(self) -> Optional[str]:
+        return self.other_device_req.eppn
+
+    @property
+    def service_info(self) -> Optional[ServiceInfo]:
+        """ Information about the service where the user is logging in """
+        return None
```

Comment:
It would be great if this could return first device service info.

Reply:
done